src/main/resources/application.yaml

```diff
@@ -0,0 +1,19 @@
+server.port: 8080
+
+bot:
+  check:
+    initialDelay: 60000
+    fixedDelay: 60000
+
+foreman:
+  baseUrl: https://dashboard.foreman.mn
+  apiUrl: https://api.foreman.mn
+
+google:
+  chatScope: https://www.googleapis.com/auth/chat.bot
+  bucket: dev-william-foreman-g-chat-bot
```

Comment:
In production app.yml, you need an environment variable:

```
google.bucket: "foreman-google-chat-bot"
```

Reply:
Ok. I was planning on letting the default named app.yaml be the production one.